From e6f13c59433db5123dc443a491ab832351a0bea8 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 2 Dec 2015 08:45:21 +0000
Subject: [PATCH 4/4] arm: machine_kexec: changed the max waiting time to
 others cpu stop

Changed the max waiting time to others cpu stop from 1000 to 2000 to
avoid below errors when running kdump:
	Non-crashing CPUs did not react to IPI

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/kernel/machine_kexec.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/kernel/machine_kexec.c b/arch/arm/kernel/machine_kexec.c
index 096b5db0..b9fc9a3 100644
--- a/arch/arm/kernel/machine_kexec.c
+++ b/arch/arm/kernel/machine_kexec.c
@@ -134,7 +134,7 @@ void machine_crash_shutdown(struct pt_regs *regs)
 
 	atomic_set(&waiting_for_crash_ipi, num_online_cpus() - 1);
 	smp_call_function(machine_crash_nonpanic_core, NULL, false);
-	msecs = 1000; /* Wait at most a second for the other cpus to stop */
+	msecs = 2000; /* Wait at most a second for the other cpus to stop */
 	while ((atomic_read(&waiting_for_crash_ipi) > 0) && msecs) {
 		mdelay(1);
 		msecs--;
-- 
1.7.5.4

